server/views/pages/article.njk

```diff
@@ -16,13 +16,7 @@
 {% endblock %}
 {% componentV2 'page-description', {title: article.headline, meta: {type: 'date', value: article.datePublished}, lead: true}, {'a': true}, {icon: article.contentType | getIconForContentType} %}
 
-{% for media in article.mainMedia %}
-  {% set seriesData = {
```

Comment:
I was just thinking this - we can reput it there in another PR as it's not used yet.